core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatterplot/ScatterplotOpExec.java

```diff
@@ -0,0 +1,44 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatterplot;
+
+import edu.uci.ics.texera.workflow.common.operators.map.MapOpExec;
+import edu.uci.ics.texera.workflow.common.tuple.Tuple;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Attribute;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema;
+import scala.Function1;
+import scala.Serializable;
+import java.util.*;
+
+
+/**
+ * Scatterplot operator to visualize the result as a scatterplot
+ *
+ */
+public class ScatterplotOpExec extends MapOpExec {
+    private final ScatterplotOpDesc opDesc;
+
+    public ScatterplotOpExec(ScatterplotOpDesc opDesc) {
+        this.opDesc = opDesc;
```

Comment:
I am not sure if we should pass the reference of Desc, or pass decoupled parameters individually. maybe @zuozhiw can confirm the style here.

Reply:
I am just following the same design as the other visualization operators

Reply:
I remember this was an old design, other visualization operators are pretty old as well. Let's wait for @zuozhiw to confirm.

Reply:
no, it is in the current master

Reply:
I know it's in the current master. But I remember this was an old design and we are changing from it. I could be wrong. So let's wait for @zuozhiw 's input...